.dockerignore

```diff
@@ -1 +1,12 @@
+.env
+.env.example
+.envrc
+.git
+.github
+.gitignore
+.golangci.yml
 helm/
```

Comment:
For the sake of consistency we should either remove the trailing / here or add it on line 12